Capstone-Projects-2022-Spring/project-uniflea
# Un 193 Query only products from same university

To test:
- sign into Uniflea using this account(email: drexelstudenttheme@gmail.com password: Kasie1234!)
- Create a product
- Confirm it appears on the home screen, and that products created in your Temple account do not appear
- sign out and into your Uniflea account
- confirm the product you created in the Drexel account does not appear on the home screen
